pkg/cmd/cli/describe/printer.go

```diff
@@ -159,6 +161,13 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 	return nil
 }
 
+// Receives a resource kind and its name, returns a formatted string with the
+// "resource_kind_alias/name", or simply "resource_name" if no alias is found
```

Comment:
Ah, I did forget to update this comment, I am now handling our short names here: https://github.com/juanvallejo/origin/blob/1ff8ec94d186fe3770c06cdece2ed8d5d3d9ef99/pkg/cmd/util/clientcmd/factory.go#L379-384